test-data/unit/check-statements.test

```diff
@@ -39,14 +39,6 @@ class A:
 [out]
 main:3: error: Return value expected
 
-[case testReturnNoneInFunctionReturningNone]
-import typing
-def f() -> None:
-    return None
-def g() -> None:
-    return f()  # E: No return value expected
```

Comment:
Can you keep the test case, but just remove the error message? I.e. it would test that this no longer generates an error message.